src/ucp/api/ucp.h

```diff
@@ -333,6 +335,19 @@ typedef enum {
 
 /**
  * @ingroup UCP_DATATYPE
+ * @brief Generate a reusable identifier for Scatter-gather IOV data type.
```

Comment:
This is the user-managed handle, to prevent repeated lookups & UMRs on the same buffer. The latter can't be satisfied by registration cache - because the memory layout may not match even if the pointer does.